scripts/test-puppeteer.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
Let's put this script in `src/e2e/puppeteer` so it's closer to the puppeteer tests.